mathics/builtin/system.py

```diff
@@ -535,7 +536,7 @@ class MemoryInUse(Builtin):
     """
     <dl>
       <dt>'MemoryInUse[]'
-      <dd>Returns the amount of memory used by the definitions object.
+      <dd>Returns the amount of memory used by the definitions object. If we can't determine this we return -1.
```

Comment:
This sounds more fluid to me:
"Returns the amount of memory used by the definitions object if we can determine it, -1 otherwise."

Reply:
I agree - thanks!  Change made in in 04fcb35a